### ISSUE TITLE
[A11Y] Very low contrast lines

### ISSUE THREAD

**Issue by bwbroersma** [bikkelhart/internet-nl#63]
Test page https://dev3.internet.nl/site/b6a.nl/31/

Note the contrast is added, but it does not have to be **3**:1, since it does not qualify under [SC 1.4.11](https://w3.org/TR/WCAG21#non-text-contrast) - [Important visual information](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). But probably some minimal contrast threshold should always hold, not sure of the value yet.

Bottom line (green-100 `#E3F0F5` to green-200 `#EEF6F9`: [**1.06**:1](https://webaim.org/resources/contrastchecker/?fcolor=E3F0F5&bcolor=EEF6F9)):
![Image](https://github.com/user-attachments/assets/e779ce2e-5d34-4f9a-a9de-69b43461696e)
https://github.com/bikkelhart/internet-nl/blob/a000e6690624e4b456fc846b74a91a4523e77771/frontend/css/result-section.css#L357

Table header bottom line (green-100 `#E3F0F5` to white-100: `#FFF`: [**1.16**:1](https://webaim.org/resources/contrastchecker/?fcolor=E3F0F5&bcolor=FFFFFF)):
![Image](https://github.com/user-attachments/assets/271ff0de-d205-4ffb-9f11-2f95f24e938b)
https://github.com/bikkelhart/internet-nl/blob/1c80edf9c94f6968bacb1f8a276cc7997db2cea7/frontend/css/result-section.css#L602

Test list item divider line (green-100 `#E3F0F5` to white-100: `#FFF`: [**1.16**:1](https://webaim.org/resources/contrastchecker/?fcolor=E3F0F5&bcolor=FFFFFF)):
![Image](https://github.com/user-attachments/assets/7186377a-9cd9-4c34-8bf9-3303365414e4)
https://github.com/bikkelhart/internet-nl/blob/87f11734d44245cdea3a41e8bfef655e83a9a49f/frontend/src/css/result-section.css#L481

Footer bottom line (transparent-300: `#0000001A` to white-100: `#FFF`: [**1.25**:1](https://webaim.org/resources/contrastchecker/?fcolor=0000001A&bcolor=FFFFFF)):
![Image](https://github.com/user-attachments/assets/4d59f9bc-8089-49fb-95e0-6741a46c294e)
https://github.com/bikkelhart/internet-nl/blob/1c80edf9c94f6968bacb1f8a276cc7997db2cea7/frontend/css/footer.css#L154

**Comment by Khdulkadir** [comment]
https://github.com/bikkelhart/internet-nl/commit/c929620392e01dc4b8b6c815aafde8250fcc6daf
